CHANGELOG.md

```diff
@@ -16,32 +28,28 @@ For example:
 }
 ```
 
-You can use the generated classes to set the width of:
-- template container
-- header container
-- footer container
+If you’re using Nunjucks, you should then add your class using the:
 
-It was already possible to set the page app width with the `$govuk-page-width` variable. This new feature is useful when creating additional custom page width classes.
+- `containerClasses` in the [page template](https://design-system.service.gov.uk/styles/page-template/#variables)
+- `containerClasses` option in the [header](https://design-system.service.gov.uk/components/header/) or [footer](https://design-system.service.gov.uk/components/footer/)
 
-- [Pull request #1626: Allow creating custom width containers and using them with template](https://github.com/alphagov/govuk-frontend/pull/1626).
-
-#### Set custom container classes on template
-
-You can now set classes on `.govuk-width-container` in the template with `containerClasses`. This is useful if you want to set a custom width class on the template container.
+Use the `$govuk-page-width` Sass variable instead if all your pages are the same width.
```

Comment:
I wonder if this is clear enough that this is _instead_ of doing all of the above? And do we think we need to front load this information?

Reply:
It's a good point and @hannalaakso and I discussed it - it probably *does* belong further up, but moving it up might make it a bit confusing because it will get in the way of the main guidance about the custom classes. So we think it's best to stick with this, but keep an eye on it and change it if any users get confused.